src/Pages/SingleReview.jsx

```diff
@@ -0,0 +1,45 @@
+import { useParams } from "react-router-dom"
+import { getComments, getReview } from "../utils/utils"
+import { useState, useEffect } from "react"
+import CommentCard from "../components/CommentCard"
+
+function SingleReview() {
+    const { review_id } = useParams()
+    const [review, setReview] = useState('')
+    const [isLoading, setIsLoading] = useState(true)
+    const [isLoading2, setIsLoading2] = useState(true);
+    const [comments, setComments]=useState('')
+    useEffect(() => {
+        getReview(review_id)
+            .then(data => {
+                setReview(data)
+                setIsLoading(false)
+            })
+    },[])
+    useEffect(() => {
+        getComments(review_id)
+        .then(data => {
+            setComments(data)
+            setIsLoading2(false)
+        })
+    },[])
+    if (isLoading || isLoading2) {
+        return <h2>Loading</h2>
+    }
+    return (
```

Comment:
At the moment when there are no comments for a review, your page says 'Comments'. Here you should indicate that there are no comments for that review. Please try to implement it.